tests/python-pytest/onnx/export/mxnet_export_test.py

```diff
@@ -238,6 +238,63 @@ def test_square():
 
     npt.assert_almost_equal(result, numpy_op)
 
+@with_seed()
+def test_greater():
```

Comment:
I am seeing that a lot of repetitions in the test code. Could you please modularize the test code? happy to discuss offline

Reply:
@nswamy @vandanavk was the concern addressed offline? what's the status of this PR?